backoffice/src/gateways/mappers/LegacyBeaconResponseMapper.ts

```diff
@@ -63,6 +62,17 @@ export class LegacyBeaconResponseMapper implements ILegacyBeaconResponseMapper {
     };
   }
 
+  private mapStatus(claimStatus: string): BeaconStatuses {
+    switch (claimStatus.trim().toUpperCase()) {
```

Comment:
Adding logic to a front end response mapper felt questionable but this is technical debt that I'm not sure we have time to sort at this stage? 😕

Reply:
```
      beaconStatus:
        beaconApiResponse.data.attributes.claimStatus === "CLAIMED"
          ? BeaconStatuses.Claimed
          : (beaconApiResponse.data.attributes.claimStatus === "DELETED"
          ? BeaconStatuses.Deleted : BeaconStatuses.Migrated),
```
          
          Not perfect, but a slight addition to what's already there. Just an option, your function works too.

Reply:
It's not poss to move this logic inside the BeaconStatuses enum 🤔